packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -482,6 +485,32 @@ class CameraController extends ValueNotifier<CameraValue> {
     }
   }
 
+  /// TODO: Documentation
+  void onCameraTimeLimitReachedEvent({onCameraTimeLimitReached}) {
+    if (!value.isInitialized || _isDisposed) {
+      throw CameraException(
+        'Uninitialized CameraController',
+        'cameraTimeLimitReachedEventStream was called on uninitialized CameraController',
+      );
+    }
+    if (!value.isRecordingVideo) {
```

Comment:
Is this correct behaviour? Should we only listen after starting the recording? Or should it be possible beforehand?